domain/src/main/java/org/oppia/domain/OnBoardingFlowController.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.domain
+
+import androidx.lifecycle.LiveData
+import org.oppia.app.model.OnBoardingFlow
+import org.oppia.data.persistence.PersistentCacheStore
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.data.DataProviders
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for persisting and retrieving the user on-boarding information of the app. */
+@Singleton
+class OnBoardingFlowController @Inject constructor(
```

Comment:
This file shouldn't be included here as it is a part of domain layer.

Reply:
This seems to be issue with Git i merged with parent branch and now not showing.